code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -45,6 +45,14 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		var/obj/item/organ/liver/L = C.getorganslot(ORGAN_SLOT_LIVER)
 		if (istype(L))
 			L.applyOrganDamage(((max(sqrt(volume) * (boozepwr ** ALCOHOL_EXPONENT) * L.alcohol_tolerance, 0))/150))
+	else
+		if(C.drunkenness > boozepwr && boozepwr <= 0)
```

Comment:
You can remove the ``C.drunkenness > boozepwr`` because the statement is always true (or at least SHOULD always be true) if boozepwr < 0.
Another thing you could optomize is to just move the ``boozepwr < 0`` to line 38 and remove the entire new block. If you do that, you can get rid of this duplicate code (generally a bad thing)

```
	if(C.drunkenness < volume * boozepwr * ALCOHOL_THRESHOLD_MODIFIER || boozepwr < 0)
```
Originally I thought that the sqrt would maybe cause issues, but that doesn't seem to be the case.

Reply:
Yep, will do thanks for showing it